pkg/game/elements/score_report.go

```diff
@@ -7,3 +7,45 @@ type ScoreReport struct {
 	// for reference, see also: player.meepleCounts
 	ReturnedMeeples map[uint8][]uint8
 }
+
+func (report *ScoreReport) JoinReport(other ScoreReport) {
```

Comment:
Please take a look at: https://github.com/YetAnotherSpieskowcy/Carcassonne-Engine/blob/d01b3f2e3d64f968c4369017b5269bfc1070ab8e/pkg/game/elements/score_report.go (score_report.go from the Scoring monasteries PR).

That version is a lot shorter and more concise and I think it has the same functionality as yours (correct me if I'm wrong though)

Reply:
i see it is not, merged already, so it won't make much difference. Simply override my version. In your version there is no checking if map field exists, or it isn't needed?

Reply:
I think it's not needed, but I'll check it later. Let's leave this as is for now